stratum/glue/init_google.h

```diff
@@ -14,13 +13,24 @@
 #if !GOOGLE_BASE_HAS_INITGOOGLE
 
 #include "gflags/gflags.h"
+#include "stratum/glue/logging.h"
 
 // TODO(unknown) need to define transformation or comment this out on
 // Google's side
 using gflags::ParseCommandLineFlags;
 
 inline void InitGoogle(const char *usage, int *argc, char ***argv,
                        bool remove_flags) {
+  // Here we can set Stratum-wide defaults or overrides for library flags.
+  CHECK(!::gflags::SetCommandLineOptionWithMode("logtostderr", "true",
+                                                ::gflags::SET_FLAGS_DEFAULT)
+             .empty());
+  CHECK(!::gflags::SetCommandLineOptionWithMode("colorlogtostderr", "true",
+                                                ::gflags::SET_FLAGS_DEFAULT)
+             .empty());
+  CHECK(!::gflags::SetCommandLineOptionWithMode("stderrthreshold", "0",
+                                                ::gflags::SET_FLAGS_DEFAULT)
+             .empty());
   ParseCommandLineFlags(argc, argv, remove_flags);
```

Comment:
For future reference: we might have to split this up in `ParseCommandLineNonHelpFlags` and `HandleCommandLineHelpFlags`. `SetCommandLineOptionWithMode` would be done in between.